### PR TITLE
bcm2835-driver: fix /opt/vc symlink creation

### PR DESCRIPTION
**Pull request by HiassofT**
On unclean builds where, the /opt/vc/ symlinks to /usr/include
and /usr/lib were already present in the sysroot, ln -sf follows
the /opt/vc symlinks and creates the symlinks in /usr/include and
/usr/lib, respectively.

Use ln -sfn instead so ln doesn't dereference existing symlinks.

ping @5schatten you added the symlinks in cd8fa8de23f

### CODE REVIEW

**Comment by MilhouseVH** [comment]
This is a repeat of what I said on Slack:

you know... the reason why we have that problem with `opt/vc` (and looking at the code, potentially some other things), is because we create the sym links in `make_target()` which is writing directly to the shared `$SYSROOT_PREFIX` - if we created the sym links (and fiddled with the `*.pc` files rewriting `/opt/vc`) in `makeinstall_target()` we'd have the luxury of creating files in the *isolated* `$SYSROOT_PREFIX` and we wouldn't be exposing the rest of the build to partially written `*.pc` files, or have the problem of existing symbolic links.

basically, just merge `make_target()` into `makeinstall_target()` and I think everything will be fine?

in general, `make_target()` (etc.) is the wrong place to mess around with the content of `$SYSROOT_PREFIX` - packages should not be modifying `$SYSROOT_PREFIX` in `make_${TARGET}`, those modifications should only happen in `makeinstall_${TARGET}`.

**Comment by 5schatten** [comment]
@MilhouseVH sounds reasonable :+1:

**Comment by HiassofT** [comment]
This needs more work and I'll put it into the safe hands of @MilhouseVH to provide a proper fix
